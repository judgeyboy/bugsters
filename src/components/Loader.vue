<template>
  <div class="bugster-loader">
    <div class="loading loading-lg"></div>
  </div>

</template>

<script>
export default {
  name: 'Loader'
}
</script>

<style lang="sass" scoped>
.bugster-loader
  min-height: 3rem
  display: flex
  justify-content: center
</style>
