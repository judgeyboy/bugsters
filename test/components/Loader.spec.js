import {shallow} from 'vue-test-utils'

import Loader from '@/components/Loader'

describe('Loader component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(Loader)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(Loader)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
