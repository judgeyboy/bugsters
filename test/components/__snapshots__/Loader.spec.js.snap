// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should match snapshot 1`] = `

<div class="bugster-loader">
  <div class="loading loading-lg">
  </div>
</div>

`;
